### DISCUSSION TITLE
Can't access Zotero Creators Fields

### DISCUSSION THREAD

**Discussion by vwrk** [retorquere/zotero-better-bibtex#2786] [Q&A]
Hello!

Please excuse me, if this is a very stupid question but I can't figure it out myself.

I am a philosophy/cultural sciences student and have a lot of entries in Zotero of texts that are published in collected writings. 

I want to configure the citation key formula in the following way:

1. One author:
AuthorYearShorttitle

2. Two authors:
Author1Author2YearShorttitle

3. Three or more authors:
Author1EtAlYearShorttitle

4. Author and editor(s) (regardless of editors of the book or the series):
AuthorYearShorttitle

5. No author, just book editor:
EditorYearShorttitle

6. No author, but book editor and series editor:
(Book)EditorYearShorttitle

Therefore, I use the following formula:
`authEtAl+year+(ShortTitle.nopunctordash.select(n=3).capitalize || shorttitle(3,3))`

In the cases 1-5 it works as expected, but in case 6 it also uses the series editor. That is quite annoying. Therefore, I tried to use the Zotero fields directly. The Zotero documentation (https://www.zotero.org/support/kb/item_types_and_fields) says it has (among others) the following fields for Item Creators: Author, Editor, Series Editor.

What I thought I could do is something like:
`(authEtAl || Editor || SeriesEditor)+year+(ShortTitle.nopunctordash.select(n=3).capitalize || shorttitle(3,3))`

When I try to use one of them in the formula it says: `Zotero items do not have a field named "Editor" at position 14`

How can I access thoese fields? In the Better BibTeX documentation under "Direct access to unprocessed fields" I couldn't find the proper entry in the table.

Thanks in advance!

All the best

**Comment by retorquere** [comment]
I don't have a simple solution for this with current BBT, but if you open an issue there's something new I'd like you to test.

**Reply by vwrk** [reply]
Hey,

thanks for your reply! I will do so!
